obol-dapp-challenge/apps/web/src/store/create-app-slice.ts

```diff
@@ -0,0 +1,6 @@
+import { asyncThunkCreator, buildCreateSlice } from '@reduxjs/toolkit';
```

Comment:
expand your choice to use redux toolkit instead of other state management libraries with less verbosity in the implemenation?